src/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://unpkg.com/boxicons@2.1.2/css/boxicons.min.css"
+      rel="stylesheet"
+    />
+    <title>LeaderBoards</title>
+  </head>
+  <body class="flex">
+    <main class="container flex">
+      <h1>
+        <i class="bx bxs-star"></i><i class="bx bxs-star"></i
+        ><i class="bx bxs-star"></i> LeaderBoard
+      </h1>
+
+      <div class="score-container flex">
+        <div class="score-placeholder border-radius">
+          <div class="score-header flex border-radius">
+            <h2>Recent scores</h2>
+            <button id="refreshBtn">Refresh</button>
+          </div>
+
+          <div class="score-display">
+            <h2 class="notif">Refresh to load data</h2>
+          </div>
+        </div>
+
+        <form>
+          <h2 class="score-header border-radius">Add Your Score</h2>
+          <input type="text" placeholder="Enter Your Name" id="name" />
+          <input type="number" placeholder="Enter Your Score" id="score" />
+          <button type="submit" id="submit">Submit</button>
```

Comment:
- [ ] Please align your "submit" button to the right so that your layout matches the wireframe 👍